GNUmakefile

```diff
@@ -1,19 +1,14 @@
-TEST?=$$(go list ./... |grep -v 'vendor')
+TEST?=$$(go list ./...)
 WEBSITE_REPO=github.com/hashicorp/terraform-website
 PKG_NAME=google
 
-GO111MODULE=on
-GOFLAGS=-mod=vendor
-
 default: build
 
 build: fmtcheck generate
 	go install
 
 test: fmtcheck generate
-	go test -i $(TEST) || exit 1
-	echo $(TEST) | \
-		xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
+	go test $(TESTARGS) -timeout=30s $(TEST)
```

Comment:
There's no need to pipe the list of packages to `xargs`. I was told parallelism was being limited to 4 for CI reasons, but I'm skeptical of that. Unless you have some very weird CI situation, limiting parallelism just slows down your tests a little. If a higher parallelism causes test failures, then you've got race conditions in your tests and they should be fixed or `parallelism` should be set to `1`.